src/main/java/com/almanza/inventorystore/models/ProductModel.java

```diff
@@ -9,7 +9,7 @@
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
-public class ProductDTO {
+public class ProductModel {
   private Integer id;
   private String brand;
   private String color;
```

Comment:
please include some property managing LocalDateTime, e.g: created
Also add an enumeration property, the enum class must have the Enum postfix